### PR TITLE
EZP-30724: Fixed message displayed when content type is locked

### PR DESCRIPTION
**Pull request by pawbuj**
| Question      | Answer
| ------------- | ---
| Tickets       | https://jira.ez.no/browse/EZP-30724
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Tests pass?   | yes
| Doc needed?   | no
| License       | [GPL-2.0](https://github.com/ezsystems/ezplatform-admin-ui/blob/master/LICENSE)

fixed message 

#### Checklist:
- [ ] Coding standards (`$ composer fix-cs`)
- [x] Ready for Code Review

### CODE REVIEW

File: src/bundle/Controller/ContentTypeController.php [hunk 1]
```diff
@@ -349,31 +349,26 @@ public function removeTranslationAction(Request $request): Response
...
-                );
+            $contentTypeDraft = $this->contentTypeService->createContentTypeDraft($contentType);
+        }
 
-                return $this->redirectToRoute('ezplatform.content_type.view', [
-                    'contentTypeGroupId' => $group->id,
-                    'contentTypeId' => $contentType->id,
-                ]);
-            }
+        if($contentTypeDraft->modifierId !== $this->getUser()->getAPIUser()->getUserId()) {
```
Comment (alongosz) [inline src/bundle/Controller/ContentTypeController.php]: You probably have CS issue here

File: src/bundle/Controller/ContentTypeController.php [hunk 2]
```diff
@@ -349,31 +349,26 @@ public function removeTranslationAction(Request $request): Response
...
+            $this->notificationHandler->error(
+                $this->translator->trans(
+                /** @Desc("Draft of the Content Type '%name%' already exists and is locked by '%userContentName%'") */
```
Comment (ViniTou) [inline src/bundle/Controller/ContentTypeController.php]: Also here prolly.

**Comment by lserwatka** [comment]
You can merge it up.

**Comment by pawbuj** [comment]
merged to master

**Pull request merged**